packages/fx-core/src/plugins/resource/bot/dotnet/plugin.ts

```diff
@@ -51,6 +52,22 @@ export class DotnetBotImpl extends TeamsBotImpl {
     return ResultFactory.Success(result);
   }
 
+  public async postLocalDebug(context: PluginContext): Promise<FxResult> {
+    await super.postLocalDebug(context);
+    const appSettingsPath = path.join(context.root, PathInfo.appSettingDevelopment);
+    try {
+      let appSettings = await fs.readFile(appSettingsPath, "utf-8");
+      const botId = AppSettingsPlaceholders.botId;
+      const botPassword = AppSettingsPlaceholders.botPassword;
+      appSettings = appSettings.replace(RegularExpr.botId, botId);
```

Comment:
Replace $botId$ with $botId$?
I think we need to read bot id from local envInfo and replace the placeholder with actual value.